wire/protobuf/generate.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 - See NOTICE file for copyright holders.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package protobuf
+
+// This command generates the bindings for serializing wire messages using
+// protocol buffers serialization protocol.
+//
+// It requires protoc and protoc-gen-go to be installed.
+
+//go:generate protoc --go_out=. wire.proto
```

Comment:
Do we document this in the readme?

Reply:
No need to do this in the general README. We also don't do it for the contract bindings.